src/content/doc-sdk-java/start.mdx

```diff
@@ -0,0 +1,318 @@
+
+```java
+// Create an empty book instance
+Book update = new Book();
+
+update.available = true;
+
+// Update all book records
+Iterator<Book> updated = driver.update(Book.class, "book", UpType.MERGE, update);
+```
+
+As you can see, we can instantiate an empty `Book` object and set only the fields we want to update. This allows us to conveniently update only specific fields.
+
+### Deleting records
+
+Existing records can be deleted using the `delete` method. You can either pass a table name to delete all records in a table, or a record ID to delete a specific record.
+
+```java
+// Delete a specific book
+driver.delete(new RecordId("book", "aeon"));
+
+// Delete all book records
+driver.delete("book");
+```
+
+### Relating records
+
+The Java SDK makes it easy to define relations between records using the `relate` method. This method takes two `RecordId` instances, an edge name, and an optional value model.
+
+This next example demonstrates how to relate a book to a publisher.
+
+```java
+RecordId bookId = new RecordId("book", "aeon");
+RecordId publisherId = new RecordId("publisher", "surrealdb");
+
+// Relate a book to a publisher
+driver.relate(bookId, "published_by", publisherId);
+```
+
+Much like other methods you can also pass a class reference as first argument to control the return and value type. This class must be a subclass of `Relation`, or simply `Relation` itself.
+This class holds the `id`, `in`, and `out` of a relation.
+
+```java
+Relation relation = driver.relate(Relation.class, bookId, "published_by", publisherId);
+            
+System.out.println("Related " + relation.in + " to " + relation.out + " with relation " + relation.id);
+```
+
+### Custom queries
+
+While the previously mentioned methods are useful for basic operations, you may want to perform more complex queries. This can be achieved using the `query` method, which allows you to execute custom SurrealQL queries.
+
+You can pass a query string to the `query` method and it will return a `Response` object containing the query results.
+
+```java
+Response response = driver.query("SELECT * FROM book WHERE available = true");
+
+// Take the result of the first statement
+Value result = response.take(0);
+```
+
+You can optionally pass parameters to the query by using the `queryBind` method, which takes a map of values to bind to the query.
+
+```java
+Response response = driver.queryBind(
+	"SELECT * FROM book WHERE available = $status",
+	Map.of("status", true)
+);
+
+// Take the result of the first statement
+Value result = response.take(0);
+```
+
+## Using values
+
+While we can represent object data using model classes, we can also use the `Value` class to represent SurrealDB data types directly.
+
+In all previous examples we can omit the model class reference arguments to make methods return a `Value` instance instead of a model class instance.
```

Comment:
In the Rust SDK using a Value is a really convenient way to avoid having to define a type to deserialize into ahead of time when first building something. Is it the same in Java? If so then maybe this could be talked up a bit more by showing all the boilerplate that it saves. (What you'd have to do if you weren't working with a Value when prototyping)